
<HTML>
<HEAD>
<TITLE>Suite Test Report</TITLE>
</HEAD>
<BODY>
<table cellspacing="1" cellpadding="2" border="1">
<tr><td>Start Time</td><td>End Time</td><td>Duration(H:M:S)</td></tr>
<tr><td>2015-11-26:16:43:04</td><td>2015-11-26:16:43:38</td><td>0:00:33.895000</td></tr>
</table>
<br>
<table cellspacing="1" cellpadding="2" border="1">
<tr><td>SUITE NAME</td><td>ARGURMENTS</td><td>CASE RANGE</td></tr>

<tr><td>st_dhcpv6.csv</td><td>E7_R02.04.050_EX24_0003,2.4.50.3</td><td>2-3</td></tr>
</table>
<br><br>

<table cellspacing="1" cellpadding="2" border="1">
<tr>
<td>TOTAL CASE</td><td bgcolor="#00FF00">PASS</td><td bgcolor="#FF0000">FAIL</td><td bgcolor="#0000FF">NOT RUN</td>
</tr>
<tr>
<td>2</td><td bgcolor="#00FF00" >0</td><td bgcolor="#FF0000">2</td><td  bgcolor="#0000FF">0</td>
</tr>
<tr>
<td> </td><td>0%</td><td>100%</td><td>0%</td>
</tr>
</table>
<BR>
<BR>
<table cellspacing="1" cellpadding="2" border="1">
<tr><td>No.</td><td>Result</td><td>Case Name</td><td>Duration(s)</td><td>StartTime</td><td>EndTime</td><td>Line No</td><td>Error Message</td></tr><tr>
        <td>1</td>
        <td bgcolor="#FF0000"><a target="+BLANK" href=".././log/st_dhcpv6-csv-2-3-E7_R02-04-050_EX24_0003-2-4-50-3-2015-11-26_164304.547000/1-2015-11-26_164304.567000">FAIL</td>
        <td><a target="+BLANK" href=".././log/st_dhcpv6-csv-2-3-E7_R02-04-050_EX24_0003-2-4-50-3-2015-11-26_164304.547000/1-2015-11-26_164304.567000">cr.py upgradeE7.csv bench.csv full N6 e7 e7 600 E7_R02.04.050_EX24_0003 2.4.50.3</td>
        <td>33.8369998932</td>
        <td>2015-11-26:16:43:04</td>
        <td>2015-11-26:16:43:38</td>
        <td>1</td>
        <td>
*   tried 1 time(s), failed in function(singleStep),
arg( ['', 'Upgrade: Had Errors', '30', False, True, False])
kwarg ({})
*
*Exception:pattern(Upgrade: Had Errors) found with 30.000000, buffer is:
*--buffer start--
*
*N6>EVENT for card 1/1 at 2015/11/26 16:48:36.99:
*    "Upgrade: Failed File Xfer --  Reason: Software package does not have files for this card"
*EVENT for system at 2015/11/26 16:48:38.49:
*    "Upgrade: Had Errors -- Version=2.4.50.3"
*
*--buffer end here--
*
*
</td>
        </tr>
<tr>
        <td>2</td>
        <td bgcolor="#FF0000"><a target="+BLANK" href=".././log/st_dhcpv6-csv-2-3-E7_R02-04-050_EX24_0003-2-4-50-3-2015-11-26_164304.547000/2-2015-11-26_164304.569000">FAIL</td>
        <td><a target="+BLANK" href=".././log/st_dhcpv6-csv-2-3-E7_R02-04-050_EX24_0003-2-4-50-3-2015-11-26_164304.547000/2-2015-11-26_164304.569000">cr.py upgradeE7.csv bench.csv full N3 e7 e7 600 E7_R02.04.050_EX24_0003 2.4.50.3</td>
        <td>33.8359999657</td>
        <td>2015-11-26:16:43:04</td>
        <td>2015-11-26:16:43:38</td>
        <td>2</td>
        <td>
*   tried 1 time(s), failed in function(singleStep),
arg( ['', 'Upgrade: Had Errors', '30', False, True, False])
kwarg ({})
*
*Exception:pattern(Upgrade: Had Errors) found with 30.000000, buffer is:
*--buffer start--
*
*N3>EVENT for card 1 at 2015/11/26 16:42:01.88:
*    "Upgrade: Failed File Xfer --  Reason: Software package does not have files for this card"
*EVENT for system at 2015/11/26 16:42:03.38:
*    "Upgrade: Had Errors -- Version=2.4.50.3"
*Alarm CLEARED for system at 2015/11/26 16:42:03.38:
*    "Software Upgrade in progress"
*
*--buffer end here--
*
*
</td>
        </tr>
</table>
<br />
<br />
</body></html>